### PR TITLE
Merge feature overrides

### PR DESCRIPTION
**Pull request by pmconne**
See NextVersion.md

### CODE REVIEW

**Review by bbastings — APPROVED** [review]

File: core/frontend/src/EmphasizeElements.ts [hunk 1]
```diff
@@ -32,17 +32,19 @@ export class EmphasizeElements implements FeatureOverrideProvider {
     const emphasizedElements = this.getEmphasizedElements(vp);
     if (undefined !== emphasizedElements) {
       overrides.setDefaultOverrides(this._defaultAppearance!);
-      const app = this.wantEmphasis ? this._emphasizedAppearance : FeatureAppearance.defaults;
-      emphasizedElements.forEach((id) => { overrides.overrideElement(id, app); });
+      const appearance = this.wantEmphasis ? this._emphasizedAppearance : FeatureAppearance.defaults;
+      for (const elementId of emphasizedElements)
+        overrides.override({ elementId, appearance });
```
Comment (kabentley) [inline core/frontend/src/EmphasizeElements.ts:37]: this is going to create an object for every iteration. Should you re-use an object?

Comment (pmconne) [reply core/frontend/src/EmphasizeElements.ts:37]: I wrote a little performance test that shows reusing an object is ~25% faster so I guess so.

Comment (pmconne) [reply core/frontend/src/EmphasizeElements.ts]: Pushed.

File: core/frontend/src/EmphasizeElements.ts [hunk 2]
```diff
@@ -32,17 +32,19 @@ export class EmphasizeElements implements FeatureOverrideProvider {
...
     const overriddenElements = this.getOverriddenElements();
     if (undefined !== overriddenElements) {
       if (undefined !== this._defaultAppearance)
         overrides.setDefaultOverrides(this._defaultAppearance);
       for (const [key, ids] of overriddenElements) {
-        const ovrApp = this.createAppearanceFromKey(key);
-        ids.forEach((id) => { overrides.overrideElement(id, ovrApp); });
+        const appearance = this.createAppearanceFromKey(key);
+        for (const elementId of ids)
+          overrides.override({ elementId, appearance });
```
Comment (kabentley) [inline core/frontend/src/EmphasizeElements.ts:52]: here too?

**Review by kabentley — APPROVED** [review]

File: core/frontend/src/EmphasizeElements.ts (hunk 2 above)
Comment (pmconne) [reply core/frontend/src/EmphasizeElements.ts]: Done.

**Comment by pmconne** [comment]
@NancyMcCallB @bsteinbk @MarcNeely @markschlosseratbentley @DStradley ui and display team review is required.

**Review by markschlosseratbentley — APPROVED** [review]

**Review by NancyMcCallB — APPROVED** [review]

**Pull request merged**